components/cam/src/physics/rrtmgp/radiation_utils.F90

```diff
@@ -302,8 +303,10 @@ subroutine handle_error(error_message, stop_on_error)
       ! nothing and return silently.
       if (len(trim(error_message)) > 0) then
          if (stop_on_error_local) then
-            call endrun(module_name // ': ' // error_message)
-         end if
+           call endrun(module_name // ': ' // error_message)
+         else
+	   write(iulog,*) 'WARNING THRESHOLD VIOLATED: ', error_message
```

Comment:
I'd prefer to change the wording here to just "WARNING: ", since this is a generic error handler that applies to situations other than just when thresholds are exceeded. For example, RRTMGP external routines return non-empty error messages when dimension sizes do not line up as well.

Reply:
Fixed.